hr_timesheet_sheet_attendance/views/hr_timesheet_sheet_view.xml

```diff
@@ -55,7 +55,7 @@
                                 <button name="attendance_action_change" attrs="{'invisible': ['|', ('attendance_state','=',False), ('attendance_state', '=', 'checked_out')]}" type="object" string="Check Out"  style="margin-left: 20px"/>
                             </div>
                         </group>
-                        <field name="attendances_ids" nolabel="1" context="{'default_employee_id': employee_id}" colspan="4">
+                        <field name="attendances_ids" nolabel="1" context="{'default_employee_id': employee_id}" colspan="4" groups="hr_attendance.group_hr_attendance_user">
```

Comment:
Maybe we should avoid the editable thing, not to hide this.

Reply:
But the manual attendance user could still edit their attendances from the attendance line form popup

Reply:
You should prevent that with no_open or readonly=1

Reply:
Ok, changes done